domain-model.md

```diff
@@ -1 +1,11 @@
 // Add your domain model below
+
```

Comment:
Really liked how you organized your domain table with "|" as mine kept formatting poorly every time

Reply:
I've used an example from the first TDD exercises' domain model. Just simplified and adapted to my needs(also was a bit lost on the columns, had to count the spaces before refactoring)